app/coffee/Features/User/UserCreator.coffee

```diff
@@ -17,23 +17,15 @@ module.exports = UserCreator =
 	createNewUser: (opts, callback)->
 		logger.log opts:opts, "creating new user"
 		user = new User()
-		user.email = opts.email
-		user.holdingAccount = opts.holdingAccount
-		user.ace.syntaxValidation = true
 
 		username = opts.email.match(/^[^@]*/)
-		if opts.first_name? and opts.first_name.length != 0
-			user.first_name = opts.first_name
-		else if username?
-			user.first_name = username[0]
-		else
-			user.first_name = ""
-
-		if opts.last_name?
-			user.last_name = opts.last_name
-		else
-			user.last_name = ""
+		if !opts.first_name? or opts.first_name == ""
+			opts.first_name = username[0]
 
+		for key, value of opts
+			user[key] = value
+			
+		user.ace.syntaxValidation = true
```

Comment:
This is just copied from the old code, but I think the fear of a default is it will get set to `true` for users that don't yet have a value (old users) if we do `user.save` somewhere else.